config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Kakama::Application.routes.draw do
+  resources :api_docs, only: [:index]
+
+  get '/api' => redirect('/swagger/dist/index.html?url=/api_docs.json')
```

Comment:
Nice work on this PR. Pretty great to have a documented json api implemented in a single PR!

My only real complaint is that I would not use `/api` for api docs, for these reasons:

* consistency - everywhere else you are using `api_docs` I believe
* clarity - without the "docs" in the route, it's not clear what the purpose the route is...
* convention - `/api` is often used as route namespace for routes, you are going against this convention which would lead to unexpected behavior for some people

So I would go with `/api-docs` personally, though you could also go with `/api_docs`. Clear what the purpose is, etc.